cnxepub/models.py

```diff
@@ -467,6 +486,58 @@ def references(self):
             return []
         return self._references
 
+    def fix_html_ids(self, mapping):
```

Comment:
Do you feel you can do that?

I suppose if we view ids as equal members of the data to that of resource links, then having it here on the model is probably alright.